src/app/components/media/SuperAdminControls.module.css

```
@@ -0,0 +1,16 @@
+.superAdminRemoveButton {
+  color: 'var(--errorMain)';
+  border-color: 'var(--errorMain)';
+  border: '2px solid';
```

```suggestion
  color: var(--errorMain);
  border-color: var(-errorMain);
  border: 2px solid;
```

This looks great Sawy! The only thing I can point out here is that the `'` quotes are not required in this file.

---

@dariusk I could not apply CSS even after remove quotes so I applied the classes to `div` not `button` and updated the custom file `SuperAdminControls.module.css`